pkg/network/placement_configuration.go

```diff
@@ -0,0 +1,67 @@
+package network
+
+import (
+	"reflect"
+
+	cnao "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/shared"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var defaultInfraNodeAffinity = corev1.NodeAffinity{
```

Comment:
My go is rusty here, but could we make it a constant?

Reply:
structs cannot be constants IUUC